src/test/java/resources/batch.feature

```diff
@@ -91,3 +91,11 @@ Feature: Batch Details API test
     Then I should get a valid token
     When I call the batch details API with expected status of 200
     Then I should get non empty response with failure and success percentage
+
+  Scenario: Batch Authorization API test
+    Given I have the demo csv file "ph-ee-bulk-demo-6.csv"
+    And I have tenant as "gorilla"
+    When I call the batch transactions endpoint with expected status of 200
+    Then I should get batchId in response
+    When I call the batch summary API with expected status of 200
```

Comment:
@arkadasfynarfin : Isn't this an async API? Shouldn't it expect 202 response with an additional assert checking whether callback URL received the callback?